docs/amber_script.md

```diff
@@ -137,18 +137,38 @@ END
 BUFFER {name} DATA_TYPE {type} {STD140 | STD430} SIZE _size_in_items_ \
     {initializer}
 
-# Defines a buffer with width and height and filled by data as specified by the
```

Comment:
Thinking about this, I think we should leave this here and just add a # Deprecated use IMAGE tag